ui/app/src/components/confirmationDialog/AnimatedConfirmation.vue

```diff
@@ -49,6 +49,8 @@
 import { defineComponent } from "vue";
 import Loader from "@/components/shared/Loader.vue";
 import SifButton from "@/components/shared/SifButton.vue";
+import { computed } from "@vue/reactivity";
```

Comment:
necessary?

Reply:
Removing. Would be nice to revisit eslint rules and have them fail builds